service/rest_version.go

```diff
@@ -19,7 +19,7 @@ import (
 	"github.com/mongodb/grip"
 	"github.com/pkg/errors"
 	"go.mongodb.org/mongo-driver/bson"
-	"gopkg.in/yaml.v3"
+	"gopkg.in/yaml.v2"
```

Comment:
Was this purposeful or should this also be `yaml "gopkg.in/20210107192922/yaml.v3"`?